sql/parser/sql.y

```diff
@@ -20,6 +20,7 @@
 
 %{
 package parser
+import "github.com/cockroachdb/cockroach/sql/privileges"
```

Comment:
is the plural necessary? `privilege{s,}.Privilege` is somehow stuttery, too.
maybe `privileges.Privilege` -> `privilege.Type`, `privilege.PrivilegeList` -> `privilege.List`?